includes/class-wc-admin-api-init.php

```diff
@@ -350,6 +352,20 @@ public static function filter_rest_endpoints( $endpoints ) {
 			$endpoints['/wc/v4/products/reviews'][1] = $endpoints['/wc/v4/products/reviews'][3];
 		}
 
+		// Override /wc/v4/settings/$group_id.
+		if ( isset( $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'] )
+			&& isset( $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][5] )
+			&& isset( $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][4] )
+			&& isset( $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][3] )
+			&& $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][3]['callback'][0] instanceof WC_Admin_REST_Setting_Options_Controller
+			&& $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][4]['callback'][0] instanceof WC_Admin_REST_Setting_Options_Controller
+			&& $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][5]['callback'][0] instanceof WC_Admin_REST_Setting_Options_Controller
+		) {
+			$endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][0] = $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][3];
+			$endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][1] = $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][4];
+			$endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][2] = $endpoints['/wc/v4/settings/(?P<group_id>[\w-]+)'][5];
+		}
+
```

Comment:
Could we skip this controller override if we weren't turning an array into a comma delimited string?

Reply:
(Answer above)